tests/checkers/inference/NninfTest.java

```diff
@@ -0,0 +1,31 @@
+package checkers.inference;
+
+import checkers.inference.solver.MaxSat2TypeSolver;
+import checkers.inference.test.CFInferenceTest;
+import org.checkerframework.framework.test.TestUtilities;
+import org.checkerframework.javacutil.Pair;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class NninfTest extends CFInferenceTest {
+
+    public NninfTest(File testFile) {
+        super(testFile, nninf.NninfChecker.class, "nninf",
+                "-Anomsgtext", "-d", "tests/build/outputdir");
+    }
+
+    @Override
+    public Pair<String, List<String>> getSolverNameAndOptions() {
+        return Pair.<String, List<String>>of(MaxSat2TypeSolver.class.getCanonicalName(), new ArrayList<String>());
+    }
+
+    @Parameters
+    public static List<File> getTestFiles() {
+        List<File> testfiles = new ArrayList<>();//InferenceTestUtilities.findAllSystemTests();
```

Comment:
Is it ok to remove the comment? What is this comment for?

Reply:
I think it's fine to remove it. I saw this comment in other subclasses of `CFInferenceTest`, so I thought it was a pattern we follow...